# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-mips.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

 0:39.23 {standard input}: Assembler messages:
 0:39.23 {standard input}:787: Error: invalid operands `lw $t4,16($4)'
 0:39.23 {standard input}:788: Error: invalid operands `lw $t5,20($4)'
 0:39.23 {standard input}:789: Error: invalid operands `lw $t6,24($4)'
 0:39.23 {standard input}:790: Error: invalid operands `lw $t7,28($4)'
 0:39.23 {standard input}:796: Error: invalid operands `sw $t4,16($5)'
 0:39.23 {standard input}:797: Error: invalid operands `sw $t5,20($5)'
 0:39.23 {standard input}:798: Error: invalid operands `sw $t6,24($5)'
 0:39.23 {standard input}:799: Error: invalid operands `sw $t7,28($5)'

--- Arctic-Fox-45.3/media/libyuv/libyuv.gyp.vanilla	2025-04-30 16:41:43.327526741 +0200
+++ Arctic-Fox-45.3/media/libyuv/libyuv.gyp	2025-04-30 16:49:17.344384246 +0200
@@ -139,7 +139,7 @@
         # Enable the following 3 macros to turn off assembly for specified CPU.
         # 'LIBYUV_DISABLE_X86',
         # 'LIBYUV_DISABLE_NEON',
-        # 'LIBYUV_DISABLE_MIPS',
+        'LIBYUV_DISABLE_MIPS',
         # This disables AVX2 (Haswell) support, overriding compiler checks
         # 'LIBYUV_DISABLE_AVX2',
         # Enable the following macro to build libyuv as a shared library (dll).
